nimble/host/mesh/src/proxy.c

```diff
@@ -1415,6 +1435,8 @@ int bt_mesh_proxy_init(void)
 	for (i = 0; i < MYNEWT_VAL(BLE_MAX_CONNECTIONS); ++i) {
 #if (MYNEWT_VAL(BLE_MESH_GATT_PROXY))
 		k_work_init(&clients[i].send_beacons, proxy_send_beacons);
+		k_delayed_work_init(&clients[i].sar_timer, proxy_sar_timeout);
+		k_delayed_work_add_arg(&clients[i].sar_timer, &clients[i]);
```

Comment:
You're adding the the sar_timer struct member unconditionally, but here you're only initialising it if BLE_MESH_GATT_PROXY is set? Isn't SAR also used for PB-GATT PDUs?

Reply:
Thank you, fixed.